mono/metadata/icall.c

```diff
@@ -8121,6 +8121,44 @@ ves_icall_System_IO_Compression_DeflateStreamNative_WriteZStream (gpointer strea
 
 #endif
 
+#if defined(TARGET_WASM) 
+
+G_EXTERN_C gpointer xamarin_timezone_get_data (MonoString* name, gint32 *size);
+G_EXTERN_C gpointer xamarin_timezone_get_names (gint32 *count);
+G_EXTERN_C MonoString* xamarin_timezone_get_local_name (void);
+
+gpointer
+ves_icall_System_TimeZoneInfo_xamarin_timezone_get_data (MonoString* name, gint32 *size)
+{
+#ifdef MONO_CROSS_COMPILE
```

Comment:
Why cross compile does not work? Put in a comment?

Reply:
Addressed in commit